test/verify/check-service

```diff
@@ -0,0 +1,129 @@
+        coverage result.
+        All json file will be saved to ".nyc_output/" folder.
+        """
+        coverage_data = self.browser.eval_js("window.__coverage__", no_trace=True)
+        str_coverage = json.dumps(coverage_data)
+        hash_str = sha256(str_coverage.encode('utf-8')).hexdigest()
+
+        cov_out_dir = Path(".nyc_output")
+        cov_out_dir.mkdir(exist_ok=True)
+
+        cov_out_file = cov_out_dir / "coverage-{}.json".format(hash_str)
+        # avoid duplecate result file
+        if not cov_out_file.is_file():
+            cov_out_file.write_text(str_coverage)
+
+    def setUp(self):
+        # remove all blueprints configured in vm.install
+        # stop and disable osbuild-composer.socket service
+        super().setUp()
+        self.allow_journal_messages("http:///run/weldr/api.socket/api/v0/.* couldn't connect: Connection refused")
+
+        self.machine.execute("""
+            for bp in $(composer-cli blueprints list); do
+                composer-cli blueprints delete $bp
+            done
+            systemctl disable --now osbuild-composer.socket
+        """)
+
+        self.addCleanup(self.machine.execute, """
+            for bp in $(composer-cli blueprints list); do
+                composer-cli blueprints delete $bp
+            done
+            for toml_file in /home/admin/files/*.toml; do
+                composer-cli blueprints push $toml_file
+            done
+        """)
+
+    def testBasic(self):
+        b = self.browser
+
+        self.login_and_go("/composer", authorized=True)
+        b.wait_present("#main")
+
+        # start and enable osbuild-composer from UI
+        b.click("button:contains('Start')")
+        # work around .blank-slate-pf element disappear in one second
+        b.wait_in_text("#main", "No Blueprints")
+        b.wait_text(".blank-slate-pf h1", "No Blueprints")
+
+        # create blueprint with random name and description
+        name_template = string.ascii_letters + string.digits + "-" + "_"
+        bp_name = "".join(random.sample(name_template, 20))
+        desc_template = string.ascii_letters + string.digits + " "
+        bp_desc = "".join(random.sample(desc_template, 40))
+        b.click(".blank-slate-pf button:contains('Create Blueprint')")
+        b.wait_present("div[role=dialog] #cmpsr-modal-crt-blueprint")
+        # blueprint name
+        b.set_input_text("#textInput-modal-markup", bp_name)
+        # blueprint description
+        b.set_input_text("#textInput2-modal-markup", bp_desc)
+        # create
+        b.click("#create-blueprint-modal-create-button")
+        b.wait_not_present("div[role=dialog] #cmpsr-modal-crt-blueprint")
+
+        # wait for all available compontents visible
+        with b.wait_timeout(300):
+            b.wait_present("ul[data-list=inputs]")
+        # search for openssh-server pacakge
+        b.set_input_text("#cmpsr-blueprint-input-filter", "openssh-server")
+        b.key_press("\r")
```

Comment:
You should be able to merge this into previous command:
`b.set_input_text("#cmpsr-blueprint-input-filter", "openssh-server\n")`

Reply:
Got it. Nice cdp feature!

Reply:
@marusak, looks `\n` not work here. https://logs.cockpit-project.org/logs/pull-931-20200416-140325-24c194ee-fedora-31-firefox/log.html

Reply:
yeah, because it is not in the `keyMap` in `test/common/testlib.py`.
But you used `\r` before, why you try to use `\n` now?

Reply:
oh because of my example. Sorry! That was just typo, `\r` is what you need.

Reply:
@marusak, `\r` does not work in this case...
https://logs.cockpit-project.org/logs/pull-931-20200417-051758-9ac4e8cc-fedora-31-firefox/log

Reply:
please take a look at definition of `set_input_text`. You will notice that it is waiting by default for the value to be the same. Which of course is not the case here, so you just need to tell it not to check the value.

Reply:
@marusak, got it. Thanks!